src/Microsoft.AspNet.Razor/CodeGenerators/CSharpTagHelperCodeRenderer.cs

```diff
@@ -659,7 +659,9 @@ private void BuildBufferedWritingScope(Chunk htmlAttributeChunk, bool htmlEncode
                 // Scopes are a runtime feature.
                 if (!_designTimeMode)
                 {
-                    _writer.WriteMethodInvocation(_tagHelperContext.StartTagHelperWritingScopeMethodName);
+                    _writer.WriteMethodInvocation(
+                        _tagHelperContext.StartTagHelperWritingScopeMethodName,
+                        "encoder: null");
```

Comment:
Ahhh, ya I wouldn't. It showcases a parameter that we're adding to fulfill an edge case. Don't want to make people think it's the right overload to use all the time.